### PR TITLE
New package: NetDocuments.ndOffice version 3.4.2.20018

### PR DESCRIPTION
**Pull request by EmmaTheEmu**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135971)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-01-24 01:40:33.766 [CLI ] ShellExecute installer failed: 1603 "Failed checking by MSO.dll file version. Any MSO.dll file locations not found in registry. Registry path: SOFTWARE\Microsoft\Windows\CurrentVersion\SharedDLLs" Action start 1:40:33: MSOfficeInsufficientVersionError. Action ended 1:40:33: MSOfficeInsufficientVersionError. Return value 3. Property(S): ErrorDialog = ErrorDlg MSI (s) (90:84) [01:40:33:561]: Product: NetDocuments ndOffice -- Installation failed. MSI (s) (90:84) [01:40:33:561]: Windows Installer installed the product. Product Name: NetDocuments ndOffice. Product Version: 3.4.2.20018. Product Language: 1033. Manufacturer: NetDocuments. Installation success or error status: 1603.

(Automated response - build 642.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 8:16:35 AM: Detected 7 WinGet errors: 8:16:35 AM: 2024-01-24 08:16:34.401 [CLI ] ShellExecute installer failed: 1603 8:16:35 AM: "Failed checking by MSO.dll file version. Any MSO.dll file locations not found in registry. Registry path: SOFTWARE\Microsoft\Windows\CurrentVersion\SharedDLLs" 8:16:35 AM: Action start 8:16:34: MSOfficeInsufficientVersionError. 8:16:35 AM: Action ended 8:16:34: MSOfficeInsufficientVersionError. Return value 3. 8:16:35 AM: Property(S): ErrorDialog = ErrorDlg 8:16:35 AM: MSI (s) (44:4C) [08:16:34:181]: Product: NetDocuments ndOffice -- Installation failed. 8:16:35 AM: MSI (s) (44:4C) [08:16:34:181]: Windows Installer installed the product. Product Name: NetDocuments ndOffice. Product Version: 3.4.2.20018. Product Language: 1033. Manufacturer: NetDocuments. Installation success or error status: 1603. 8:16:35 AM: Detected 0 Manual Dependency: 8:16:35 AM: 8:16:35 AM: Detected 0 PowerShell errors: 8:16:35 AM: Detected 1 Application Log errors: 8:16:35 AM: Product: NetDocuments ndOffice -- Unable to check current Office version, please repair your Office installation and try again.

(Automated response - build 629.)

**Comment by EmmaTheEmu** [comment]
"Product: NetDocuments ndOffice -- Unable to check current Office version, please repair your Office installation and try again"

Seems like the automatic test doesn't have office installed and that's why the installer fails.
Is there anything I need to modify to include it?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-01-24 01:40:33.766 [CLI ] ShellExecute installer failed: 1603 "Failed checking by MSO.dll file version. Any MSO.dll file locations not found in registry. Registry path: SOFTWARE\Microsoft\Windows\CurrentVersion\SharedDLLs" Action start 1:40:33: MSOfficeInsufficientVersionError. Action ended 1:40:33: MSOfficeInsufficientVersionError. Return value 3. Property(S): ErrorDialog = ErrorDlg MSI (s) (90:84) [01:40:33:561]: Product: NetDocuments ndOffice -- Installation failed. MSI (s) (90:84) [01:40:33:561]: Windows Installer installed the product. Product Name: NetDocuments ndOffice. Product Version: 3.4.2.20018. Product Language: 1033. Manufacturer: NetDocuments. Installation success or error status: 1603.

(Automated response - build 647.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 1:17:04 PM: Product: NetDocuments ndOffice -- Please install the Visual Studio 2010 Tools for Office Runtime then run this installer again. You may download the latest version from the Microsoft Download Center: http://go.microsoft.com/fwlink/?LinkId=140384

(Automated response - build 643.)

**Comment by stephengillie** [comment]
This appears to be `Microsoft.VSTOR`. 

```PowerShell
Name                                                  Id              Version       Source
----                                                  --              -------       ------
Microsoft Visual Studio 2010 Tools for Office Runtime Microsoft.VSTOR 10.0.60912.00 winget
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-01-24 01:40:33.766 [CLI ] ShellExecute installer failed: 1603 "Failed checking by MSO.dll file version. Any MSO.dll file locations not found in registry. Registry path: SOFTWARE\Microsoft\Windows\CurrentVersion\SharedDLLs" Action start 1:40:33: MSOfficeInsufficientVersionError. Action ended 1:40:33: MSOfficeInsufficientVersionError. Return value 3. Property(S): ErrorDialog = ErrorDlg MSI (s) (90:84) [01:40:33:561]: Product: NetDocuments ndOffice -- Installation failed. MSI (s) (90:84) [01:40:33:561]: Windows Installer installed the product. Product Name: NetDocuments ndOffice. Product Version: 3.4.2.20018. Product Language: 1033. Manufacturer: NetDocuments. Installation success or error status: 1603.

(Automated response - build 649.)

File: manifests/n/NetDocuments/ndOffice/3.4.2.20018/NetDocuments.ndOffice.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: NetDocuments.ndOffice
+PackageVersion: 3.4.2.20018
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{9A370100-73F5-4D5B-8E24-98367D0CFB19}'
+Installers:
```
Comment (stephengillie) [inline manifests/n/NetDocuments/ndOffice/3.4.2.20018/NetDocuments.ndOffice.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.Office
Installers:
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**